fuzzers/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+# Ensure that MinGW provides the correct header files.
+IF (WIN32 AND NOT CYGWIN)
+	ADD_DEFINITIONS(-DWIN32 -D_WIN32_WINNT=0x0600)
```

Comment:
Can we just move this into the top-level `CMakeLists.txt`?

Reply:
Good idea, done!

Reply:
One minor difference is that this is now tucked inside an [ELSE](https://github.com/libgit2/libgit2/pull/5054/files#diff-af3b638bc2a3e6c650974192a53c7291R202) block so it will only execute if we're not building with [MSVC](https://github.com/libgit2/libgit2/pull/5054/files#diff-af3b638bc2a3e6c650974192a53c7291R122).  

I _think_ that we want to continue setting this value on all platforms, when we're using MSVC, MinGW or Cygwin...  Does that sound correct?

Reply:
I moved it out of the `ELSE` block for MSVC (not sure if it makes a difference on MSVC). As before, `_WIN32_WINNT` is not set on Cygwin, I believe Cygwin doesn't provide Windows header files anyway.